examples/Capi.cpp

```diff
@@ -201,16 +202,46 @@ void contents()
     }
 
 //////6. Animation with a picture
-    animation = tvg_animation_new();
-    Tvg_Paint* pict_lottie = tvg_animation_get_picture(animation);
+    //load from a file
+    animation1 = tvg_animation_new();
+    Tvg_Paint* pict_lottie = tvg_animation_get_picture(animation1);
     if (tvg_picture_load(pict_lottie, EXAMPLE_DIR"/lottie/sample.json") != TVG_RESULT_SUCCESS) {
-        printf("Problem with loading an lottie file\n");
-        tvg_animation_del(animation);
+        printf("Problem with loading a lottie file from a file\n");
+        tvg_animation_del(animation1);
+        animation1 = NULL;
     } else {
         tvg_paint_scale(pict_lottie, 3.0f);
         tvg_canvas_push(canvas, pict_lottie);
     }
 
+    //load from a memory
+    FILE *file = fopen(EXAMPLE_DIR"/lottie/sample.json", "r");
+    if (file == NULL) return;
+    fseek(file, 0, SEEK_END);
+    long data_size = ftell(file);
+    fseek(file, 0, SEEK_SET);
+    auto data = (char*)malloc(data_size + 1);
+    if (!data) return;
+    if (fread(data, 1, data_size, file) != (size_t)data_size) {
+        free(data);
+        fclose(file);
+        return;
+    }
+    data[data_size++] = '\0';
+
+    animation2 = tvg_animation_new();
+    pict_lottie = tvg_animation_get_picture(animation2);
+    if (tvg_picture_load_data(pict_lottie, data, data_size, nullptr, nullptr, false) != TVG_RESULT_SUCCESS) {
```

Comment:
Suggestion: we could replace the original animation sample with this one and keep the API usage as a comment.
```
//You can directly load an animation from a file
//if (tvg_picture_load(pict_lottie, EXAMPLE_DIR"/lottie/sample.json") != TVG_RESULT_SUCCESS) {
```